services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/utils/IOfFlowDumpProducer.java

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2019 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.floodlight.utils;
+
+import org.projectfloodlight.openflow.protocol.OFFlowStatsEntry;
+import org.projectfloodlight.openflow.types.DatapathId;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+public interface IOfFlowDumpProducer {
```

Comment:
As per OpenKilda code convention (which is based on Google Java Style), special prefixes or suffixes are NOT used. Please, reconsider the usage of  "I" prefix here.

Reply:
It means, we should tune style check to deny such names. :) I will rename it, but probably not in this fix.